src/wiki/forms.py

```diff
@@ -399,7 +399,7 @@ def __init__(self, request, urlpath_parent, *args, **kwargs):
     slug = WikiSlugField(
         label=_('Slug'),
         help_text=_(
-            "This will be the address where your article can be found. Use only alphanumeric characters and - or _. Note that you cannot change the slug after creating the article."),
+            "This will be the address where your article can be found. Use only alphanumeric characters and - or _.<br>Note: When changing the slug later links pointing to this article are not updated."),
```

Comment:
Is this text marked safe? Will the `<br>` not be escaped? 

Suggesting to change latter part to: *Note: After changing the slug, links pointing to this article are NOT updated.*